src/steamcompmgr.cpp

```diff
@@ -395,23 +396,24 @@ static inline void stats_printf( const char* format, ...)
 	}
 }
 
+uint64_t get_time_in_nanos()
+{
+	timespec ts;
+	clock_gettime(CLOCK_MONOTONIC_RAW, &ts);
+	return ts.tv_sec * 1'000'000'000ul + ts.tv_nsec;
+}
+
 unsigned int
 get_time_in_milliseconds (void)
 {
-	struct timeval  tv;
-
-	gettimeofday (&tv, NULL);
-	return tv.tv_sec * 1000 + tv.tv_usec / 1000;
+	return (unsigned int)(get_time_in_nanos() / 1'000'000ul);
 }
 
-struct timeval lastvblankmsgtv;
-std::mutex lastvblankmsgtv_lock;
+std::atomic<uint64_t> g_lastvblank;
```

Comment:
Not related to this PR per se, but maybe we can make this `static`? Would be nice to also init this to a sane-ish value, like 0.

Reply:
Sure, that's reasonable.